### ISSUE TITLE
startRecorder failing silently - no recording possible

### ISSUE THREAD

**Issue by Boehrsi** [dooboolab/flutter_sound#192]
### Version of flutter_sound
2.0.0

## flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, v1.12.13+hotfix.5, on Linux, locale de_DE.UTF-8)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[✓] Android Studio (version 3.5)
[✓] Connected device (1 available)

• No issues found!

### Platforms you faced the error (IOS or Android or both?)
Tested only on Android.

### Expected behavior
Calling startRecorder starts the recording.

### Actual behavior
Failing silently, no recording is started.
1. Method channel call for `isEncoderSupported` comes in
2. Method channel call for `startRecorder` comes in
3. Execution is made and https://github.com/dooboolab/flutter_sound/blob/master/android/src/main/java/com/dooboolab/fluttersound/FlutterSoundPlugin.java#L265 is triggered
4. Even though there should be a result in the Dart part, as send by FlutterSoundPlugin.java line 265, I receive nothing in Dart. Found the actual problem with Java breakpoints.

### Side note
Everything worked well with previous versions (e.g.  ^1.5.0)

### Tested environment (Emulator? Real Device?)
- Android API 28 (Emulator)
- Android 8.0 (Samsung S7)

### Steps to reproduce the behavior
Execute:
```void startRecorder() async {
    try {
      print("Starting recording...");
      String savePath = await _flutterSound.startRecorder(null);
      print("Path to recording: " + savePath);
    } catch (err) {
      print('startRecorder error: $err');
    }
```

**Comment by Boehrsi** [comment]
First broken version is 1.8.0.

**Comment by hyochan** [comment]
Fixed in `2.0.1`. Just try `startRecorder()` after updating.
